website/docs/commands/compile.md

```diff
@@ -28,31 +30,47 @@ For a full list of options, run `scala-cli compile --help`, or check the options
 
 `--test` makes `scala-cli` compile main and test scopes:
 
-```bash ignore
-scala-cli compile --test Hello.scala
+```scala title=Sample.test.scala
+//> using lib "org.scalameta::munit:0.7.29"
```

Comment:
IMO we miss a mechanism to update libraries into a documentation. 

Perhaps `dependency-update` command will work fine with markdown files, but it is out of the scope this PR.

Reply:
https://github.com/VirtusLab/scala-cli/issues/1619